src/scripts/commands/dataValues.ts

```diff
@@ -168,3 +175,83 @@ const postDanglingValuesCmd = command({
         new PostDanglingValuesUseCase(dataValuesRepository, danglingDataValuesRepository).execute(args);
     },
 });
+
+const monitoringDataValues = command({
+    name: "monitoring-values",
+    description:
+        "Monitoring data values and sending an email depending on how long has passed since the last Updated",
+    args: {
+        url: getApiUrlOption(),
+        storage: option({
+            type: oneOf(["datastore", "json"]),
+            long: "storage",
+            description: "datastore or json",
+        }),
+        dataStoreNameSpace: option({
+            type: string,
+            long: "datastore-namespace",
+            defaultValue: () => "namespace to be used ",
+        }),
+        jsonSettingsPath: option({
+            type: string,
+            long: "json-settings-path",
+            description: "Path to json file that contains the dataset settings",
+            defaultValue: () => "",
```

Comment:
[subjective] I prefer to set the optional al optional: `type: optional(string)` and remove a dubious empty string as default value.